_studio/mfx_lib/decode/h265/src/mfx_h265_dec_decode.cpp

```diff
@@ -616,14 +581,12 @@ mfxStatus VideoDECODEH265::DecodeHeader(VideoCORE *core, mfxBitstream *bs, mfxVi
 
     if (umcRes == UMC::UMC_ERR_NOT_ENOUGH_DATA)
```

Comment:
MFX_CHECK

Reply:
I don't want to see it in trace, it's not an error actualy

Reply:
agree